azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -134,6 +134,16 @@ func resourceArmMsSqlDatabase() *schema.Resource {
 				ValidateFunc:     validation.IsRFC3339Time,
 			},
 
+			"recoverable_database_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"restorable_dropped_database_id": {
```

Comment:
Would this make more sense as
```suggestion
			"recover_dropped_database_id": {
```

Reply:
`recoverable` and `restorable` databases are two kinds of database, they are restored by `restore` and `recovery` two create mode. The difference, from my aspect, is that if you have deleted a db, you could immediate `restore` it. If it's too long time to be got from `restorable` dbs, you could get it from `recoverable` dbs and `recovery` it. I'd like to suggest not to change the name to this to cause confusions.

Reply:
ahh ok that makes sense, then this should be
```suggestion
			"restore_dropped_database_id": {
```

Reply:
This still hasn't been changed to 
```suggestion
			"restore_dropped_database_id": {
```